src/modules/book-favorite/book-favorite.controller.ts

```diff
@@ -0,0 +1,33 @@
+import { Body, Controller, Delete, Get, Param, Post } from '@nestjs/common';
+import { BookFavoriteService } from './book-favorite.service';
+import { BookFavoriteItemDto } from 'src/modules/book-favorite/dto/book-favorite-item.dto';
+import { BookFavoriteCreateDto } from 'src/modules/book-favorite/dto/book-favorite-create.dto';
+import type { Uuid } from 'src/common/type';
+
+@Controller('book-favorites')
+export class BookFavoriteController {
+  constructor(private readonly bookFavoriteService: BookFavoriteService) {}
+
+  @Get()
+  async findAll(): Promise<BookFavoriteItemDto[]> {
+    return BookFavoriteItemDto.fromBookFavorites(
+      await this.bookFavoriteService.findAll(),
+    );
+  }
+
+  @Post()
+  async create(
+    @Body() bookFavoriteCreateDto: BookFavoriteCreateDto,
+  ): Promise<BookFavoriteItemDto> {
```

Comment:
Không quá cần thiết phải trả về response ở endpoint này


```suggestion
  ): Promise<void> {
```